wfl/Controllers/RestaurantsController.cs

```diff
@@ -29,7 +29,7 @@ public RestaurantsController(IDBManager dbManager)
 
         [Route("{id}")]
         [HttpPut]
-        public async Task<IHttpActionResult> UpdateAsync([FromUri] string id, [FromBody] RestaurantDto restaurant)
+        public async Task<IHttpActionResult> UpdateAsync([FromBody] RestaurantDto restaurant, [FromUri] string id)
```

Comment:
I reversed the order of these arguments because I thought it was the source of an error inside the hypermedia profile, but it wasn't. When I fixed the problem, I forgot to change them back. I'll change the order back to what it was.